### ISSUE TITLE
Is there a sample code of how to use nuget command ?

### ISSUE THREAD

**Issue by Yvees** [zabirauf/icsharp#61]
hi guys,
Im new to icsharp. Is that possible using a nuget command to add references ?

**Comment by zabirauf** [comment]
@KevinYeti Yep you can. You can get the DLLs using nuget and then refer to them in Jupyter using

```
#r 'path/to/module.dll'

using moduleNS.module;
```

**Comment by Yvees** [comment]
> @KevinYeti Yep you can. You can get the DLLs using nuget and then refer to them in Jupyter using
> 
> ```
> #r 'path/to/module.dll'
> 
> using moduleNS.module;
> ```

Thank you for reply. 
Actually  what i mean is is that a way to add nuget package reference. 
maybe something like this ? 
```
#r  nuget:Json.Net
```
